Tests/SwiftLintFrameworkTests/ExplicitInitRuleTests.swift

```diff
@@ -13,10 +13,24 @@ class ExplicitInitRuleTests: SwiftLintTestCase {
             Example("foo(↓.init())")
         ]
 
+        var correction = [
+            Example("""
+            f { e in
+                A↓.init(e: e)
+            }
```

Comment:
Could you also add a comment to show that it is neither removed:
```suggestion
            f { e in
                // comment
                A↓.init(e: e)
            }
```

Reply:
I think there is no need to add special comments here. It's fine as long as future tests go well.

Reply:
The comment is not supposed to serve documentation purposes but to ensure that comments don't get lost either when the rule rewrites code.

Reply:
Oh, understood what you mean now. Added it.